tests/rptest/services/redpanda.py

```diff
@@ -394,14 +394,26 @@ def registered(self, node):
         return broker is not None
 
     def controller(self):
-        kc = KafkaCat(self)
-        cid = kc.metadata()["controllerid"]
-        self.logger.debug("Controller reported with id: {}".format(cid))
```

Comment:
+1 the amount of noise in ducktape logs that will be reduced is significant, too.